### ISSUE TITLE
@noinspection ParameterDefaultValueIsNotNullInspection not working

### ISSUE THREAD

**Issue by Dragonrun1** [kalessil/phpinspectionsea#1293]
Php Inspections (EA Extended) 
PHP 7.2

![NoInspectionNotNull](https://user-images.githubusercontent.com/1431541/56088791-78d84980-5e3c-11e9-9711-e6f0af1a1a62.PNG)
Inspection not suppressed when no inspection used. The inspection should be suppressed. Before or after method docblock does seem to matter either.

**Comment by orklah** [comment]
Hi,

It seems that PhpStorm add the docblock here automatically:
```php
class foo
{
    function bar(/** @noinspection ParameterDefaultValueIsNotNullInspection */ $a = 'a')
    {

    }
}
```

And when it is placed here, it supress the inspection.

Not sure if it is wanted or not though...

**Comment by kalessil** [comment]
Snap, it also cannot be null due to type int.
PS: suppression is completely managed by IDE,.

**Comment by Dragonrun1** [comment]
Having to put the Docblock inside the param list seems like an anti-pattern/bad practice to me.

**Comment by orklah** [comment]
> Having to put the Docblock inside the param list seems like an anti-pattern/bad practice to me.

I agree, but as kalessil said, this part is managed by PhpStorm. You should create an issue here to suggest a change: https://youtrack.jetbrains.com/issues/

**Comment by kalessil** [comment]
@Dragonrun1: agree with you, also @orklah proposed a good thing: PhpStorm backlog has similar bug-reports already, so voting totally makes sense.

**Comment by kalessil** [comment]
Fixed! @Dragonrun1, if there are other inspections which you have to suppress, a feedback-ticket is appreciated - I can at least revisit the inspections with the fresh look for false-positives, new settings or dropping them completely.
